treeherder/webapp/wsgi.py

```diff
@@ -18,7 +18,9 @@
 
 """
 import os
+from django.conf import settings
```

Comment:
Yeah I was looking at that - I think we can drop that entirely, since it's only used for mod_wsgi with multiple sites.